rb/lib/selenium/webdriver/common/error.rb

```diff
@@ -60,57 +70,58 @@ class StaleElementReferenceError < WebDriverError; end # 10
       class ElementNotVisibleError < WebDriverError; end # 11
 
       #
-      # Raised when an interaction could not be performed because the element
-      # is in an invalid state (e.g. attempting to click a disabled element).
+      # A command could not be completed because the element is in an
+      # invalid state, e.g. attempting to click an element that is no
```

Comment:
@twalpole thanks for catching this, it's been declared a bug in w3c.
@jaysonesmith we'll update the text when the w3c language is approved/updated. (https://github.com/w3c/webdriver/pull/990)